libtock/services/alarm.h

```diff
@@ -109,19 +127,19 @@ int libtock_alarm_in_ms(uint32_t ms, libtock_alarm_callback cb, void* opaque, li
  * the repeating alarm is outstanding.
  *
  * \param ms the interval to fire the alarm at in milliseconds.
- * \param callback a callback to be invoked when the alarm expires.
+ * \param cb a callback to be invoked when the alarm expires.
  * \param opaque pointer passed to the callback.
- * \param a pointer to a new alarm_repeating_t to be used by the implementation to
+ * \param alarm_repeating pointer to a new libtock_alarm_t to be used by the implementation to
  *        keep track of the alarm.
  */
-void libtock_alarm_repeating_every(uint32_t ms, libtock_alarm_callback cb, void* opaque,
-                                   libtock_alarm_repeating_t* alarm_repeating);
+void libtock_alarm_repeating_every_ms(uint32_t ms, libtock_alarm_callback cb, void* opaque,
+                                      libtock_alarm_t* alarm);
 
 /** \brief Cancels an existing repeating alarm.
```

Comment:
This comment is now outdated, since this function is used to cancel repeating alarms or any alarm set in milliseconds.